src/main/java/nusemp/logic/commands/event/EventLinkCommand.java

```diff
@@ -25,20 +27,29 @@ public class EventLinkCommand extends Command {
     public static final String COMMAND_WORD = "link";
 
     public static final String MESSAGE_USAGE = CommandType.EVENT + " " + COMMAND_WORD
-            + ": Links a contact to an event.\n\n"
+            + ": Links a contact or all filtered contacts to an event. \n\n"
             + "Parameters: "
             + PREFIX_EVENT + " EVENT_INDEX "
-            + PREFIX_CONTACT + " CONTACT_INDEX\n"
+            + PREFIX_CONTACT + " CONTACT_INDEX or 'all'\n"
             + "Example: " + CommandType.EVENT + " " + COMMAND_WORD + " "
             + PREFIX_EVENT + " 1 "
-            + PREFIX_CONTACT + " 2";
+            + PREFIX_CONTACT + " 2\n"
+            + "Example: " + CommandType.EVENT + " " + COMMAND_WORD + " "
+            + PREFIX_EVENT + " 1 "
+            + PREFIX_CONTACT + " all\n\n"
+            + "Note: EVENT_INDEX and CONTACT_INDEX must be a positive integer within the size of the displayed "
```

Comment:
```suggestion
            + "Note: EVENT_INDEX and CONTACT_INDEX must be positive integers within the size of the displayed "
```
Also, could you edit the event rsvp message and include this also?

Reply:
Added in [cdd9df8](https://github.com/AY2526S1-CS2103T-F15b-2/tp/pull/196/commits/cdd9df82e6e988fd25e7dc39fa3ca6fac2b94e81)